include/core/tag.h

```diff
@@ -110,6 +110,15 @@ int lf_tag_compare(tag_t tag1, tag_t tag2);
  */
 tag_t lf_delay_tag(tag_t tag, interval_t interval);
 
+/**
+ * Return the latest tag that cannot have an event provided that
```

Comment:
How about something like:
"Given a `next_event_tag` of an upstream event producer and a `delay` associated the connection to that event producer, return the largest tag smaller than `next_event_tag + delay` for which input from the producer can be assumed to be absent."

The name of the function could then be `lf_largest_tag_smaller_than_sum` or something like that...

Reply:
Is there actually any significance to the absence of input? Could it just be described as a numerical operation on tags? "Get the largest tag smaller than the sum of the given `tag` and `delay`?"

Reply:
That sounds like a good solution!

Reply:
I have changed lf_delay_antitag to lf_delay_strict and adjusted the implementation and the documentation. I believe this addresses the concerns and gives a simpler interpretation.